src/CommandLineParser.ts

```diff
@@ -38,6 +38,11 @@ const optionDeclarations: CommandLineOption[] = [
         type: "enum",
         choices: Object.values(LuaTarget),
     },
+    {
+        name: "noImplicitSelf",
+        description: "Implicitly enable @noSelfInFile for all target files.",
```

Comment:
I think it would be nice to describe what it actually does and keep `--help` useful without making user look up for annotations documentation